docs/photoniq/vwrs/troubleshoot-vwrs.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 120
+title: Troubleshoot VWRs
+---
+
+This page provides troubleshooting information for Macrometa PhotonIQ Virtual Waiting Rooms (VWRs).
+
+## No Waiting Room Configured
+
+If a waiting room has not been configured, then the waiting room library returns the header _No-Waiting-Room_, which indicates no waiting room was configured. This means that the domain URL is not configured correctly.
+
+## Troubleshoot Waiting Room HTML
+
+You can configure the cloud origin (e.g. NetStorage) path to the waiting room HTML through the library or the domain record. To preview this HTML, open your preferred browser and navigate to the following URL: `https://<hostname>/<waiting-room-path>/preview`
```

Comment:
I am confused on this.  "cloud origin path" and the netstorage reference.
@rahul-naik1 The last sentence makes sense.  But the first part of the sentence is confusing. 
"... the waiting room HTML through the library or domain record." makes sense.  But I am struggling with the first part of the sentence.  Can you clarify this part...

Reply:
The cloud origin path is a cloud storage path (e.g. NetStorage or S3) where the waiting room HTML page is stored.
We can connect to discuss this.